stackimage/Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM adoptopenjdk/openjdk11-openj9:ubi AS maven
+ARG BASE_IMAGE=adoptopenjdk/openjdk11-openj9:ubi
```

Comment:
A step in the right direction, but without the ability to customize the actual build command in say CP4Apps, this is just a piece of the puzzle.